bin/list-all

```diff
@@ -12,12 +12,11 @@ cmd="$cmd $releases_path"
 function sort_versions() {
     sed 'h; s/[+-]/./g; s/.p\([[:digit:]]\)/.z\1/; s/$/.z/; G; s/\n/ /' \
     | LC_ALL=C sort -t. -k 1,1 -k 2,2n -k 3,3n -k 4,4n -k 5,5n \
-    | awk '{print $2}' \
-    | paste -s -d" " -
+    | awk '{print $2}' 
 }
 
 function extract_versions_numbers() {
-    grep -o "name\": \"[0-9]\+.[0-9]\+.[0-9]\+" | sed 's/name": "//'
+    grep -o "name\": \".*" | sed 's/name": "\(.*\)\",/\1/'
```

Comment:
Right now with this regex we are _grepping_ **everything**, which could return false positives

![Screenshot 2019-06-18 20 03 41](https://user-images.githubusercontent.com/14015177/59725488-2cc2d000-9204-11e9-97f0-d76acdefdad2.png)

WDYT about just modifying the regex to additionally match the release name if there is any? 
I think something like `grep -o "name\": \"[0-9]\+.[0-9]\+.[0-9]\+-\?\w\+"` would work

![Screenshot 2019-06-18 20 01 39](https://user-images.githubusercontent.com/14015177/59725456-17e63c80-9204-11e9-9cc2-57ac64d20baa.png)

Reply:
I don't think it matters too much, but I think it's a matter of what what tradeoff's you want to take.

I'll make the change.